commands/publish/command.js

```diff
@@ -110,6 +110,15 @@ exports.builder = (yargs) => {
       describe: "Verify package read-write access for current npm user.",
       type: "boolean",
     },
+    "summary-file": {
+      // generate lerna publish json output.
+      type: "string",
```

Comment:
Please add back the describe (and make it match what you have in `lerna-schema.json`)

Reply:
resolved in b0a9cc7